src/util/tubemap.js

```diff
@@ -511,17 +511,22 @@ function createTubeMap() {
   alignSVG(nodes, tracks);
   defineSVGPatterns();
 
-  drawTrackRectangles(trackRectangles);
-  drawTrackCurves();
-  drawReversalsByColor(trackCorners, trackVerticalRectangles);
-  drawTrackRectangles(trackRectanglesStep3);
-  drawTrackRectangles(trackRectangles, "read");
-  drawTrackCurves("read");
+  // all drawn tracks are grouped 
+  let trackGroup = svg.append("g").attr("class", "track")
+  drawTrackRectangles(trackRectangles, undefined, trackGroup);
+  drawTrackCurves(undefined, trackGroup);
+  drawReversalsByColor(trackCorners, trackVerticalRectangles, undefined, trackGroup);
+  drawTrackRectangles(trackRectanglesStep3, undefined, trackGroup);
```

Comment:
I think the `typeof type === "undefined"` checks were meant to check if the argument was omitted or not. So turning around and manually passing `undefined` changes what the code at the other end really means in a needlessly complicated way.

It would make more sense to leave the optional argument last and make the new argument a next-to-last argument, or else to explicitly pass the type here instead of having the argument be optional.